playbooks/_buildapp.yml

```diff
@@ -0,0 +1,20 @@
+---
+
+- name: Build Application
+  hosts: fpm[0]
```

Comment:
Good point.  Maybe we'll need to variable-ize the "worker" group name, which this case is fpm.